app/app.js

```diff
@@ -24,6 +24,12 @@ const Menu = remote.Menu;
 const MenuItem = remote.MenuItem;
 
 const tinycolor = require('tinycolor2');
+const nunjucks = require('nunjucks');
```

Comment:
oh sorry I just mean if an engineer works with multiple programming languages that use material, it might be nice to do:

[
  {
    format: "$material-color('$HUE', '$VALUE');",
    transform: "-x",
  },
  {
    format: "MATERIAL_COLOR_$HUE_$VALUE",
    transform: "_X",
  }
]